lib/global_id.rb

```diff
@@ -1,9 +1,14 @@
 require 'global_id/global_id'
+require 'active_support'
 
 autoload :SignedGlobalID, 'global_id/signed_global_id'
```

Comment:
Agree.
But `SignedGlobalID` is defined at Top level. Is eager load usable in that case?